packages/itwin/tree-widget/src/components/trees/VisibilityTreeEventHandler.ts

```diff
@@ -133,6 +137,7 @@ export class VisibilityTreeEventHandler extends UnifiedSelectionTreeEventHandler
   }
 
   public override onCheckboxStateChanged(event: TreeCheckboxStateChangeEventArgs) {
+    this._reportUsage?.({ featureId: "visibility-change", reportInteraction: false });
```

Comment:
Maybe this could be done in checkbox component?

Reply:
Moved to `VisibilityTreeNodeCheckbox`.